source/crud/tests/unified/bulkWrite-deleteMany-let.yml

```
@@ -0,0 +1,92 @@
+description: "BulkWrite deleteMany-let"
+
+schemaVersion: "1.0"
+
+createEntities:
+  - client:
+      id: &client0 client0
+      observeEvents: [ commandStartedEvent ]
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: &database0Name crud-tests
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: &collection0Name coll0
+
+initialData: &initialData
```

I'm not sure if this was copied from elsewhere, but there's no reason to create a YAML anchor if we're not actually going to refer to it later with an alias. A quick grep of this PR didn't turn up anything for `*initialData` so I think this can be removed.

```suggestion
initialData:
```